python/grass/jupyter/interactivemap.py

```diff
@@ -764,8 +764,8 @@ def __init__(
         self._ipywidgets = ipywidgets
         self.raster_name = rasters
         self.vector_name = vectors
-        self.width = width
         self.query_control = None
+        self.width = width
```

Comment:
This looks like a leftover change that is not needed.

Reply:
I doubled checked and it looks like this is needed. The width is getting pass in by `InteractiveMap._create_toggle_button.` when building the `InteractiveQueryController`.

Reply:
Right, but it's already there, you are just moving the line.